<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Exercises: Inspect Specific GEO Series | Finding Expression Data with GEOmetadb</title>
  <meta name="description" content="<p>A hands-on tutorial that shows how to explore GEO metadata with GEOmetadb,
ask reproducible questions in SQL from R, and shortlist RNA-seq series.</p>" />
  <meta name="generator" content="bookdown 0.46 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Exercises: Inspect Specific GEO Series | Finding Expression Data with GEOmetadb" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="<p>A hands-on tutorial that shows how to explore GEO metadata with GEOmetadb,
ask reproducible questions in SQL from R, and shortlist RNA-seq series.</p>" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Exercises: Inspect Specific GEO Series | Finding Expression Data with GEOmetadb" />
  
  <meta name="twitter:description" content="<p>A hands-on tutorial that shows how to explore GEO metadata with GEOmetadb,
ask reproducible questions in SQL from R, and shortlist RNA-seq series.</p>" />
  

<meta name="author" content="Ruth Isserlin" />


<meta name="date" content="2026-01-28" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="finding-datasets-of-interest.html"/>
<link rel="next" href="next-week-next-steps.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<!-- custom header -->


<style type="text/css">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">GEOmetadb Tutorial</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Welcome</a></li>
<li class="chapter" data-level="1" data-path="overview.html"><a href="overview.html"><i class="fa fa-check"></i><b>1</b> Overview</a></li>
<li class="chapter" data-level="2" data-path="geo-datatypes.html"><a href="geo-datatypes.html"><i class="fa fa-check"></i><b>2</b> GEO Datatypes</a></li>
<li class="chapter" data-level="3" data-path="setting-up-geometadb.html"><a href="setting-up-geometadb.html"><i class="fa fa-check"></i><b>3</b> Setting up <code>GEOmetadb</code></a>
<ul>
<li class="chapter" data-level="3.1" data-path="setting-up-geometadb.html"><a href="setting-up-geometadb.html#install-packages"><i class="fa fa-check"></i><b>3.1</b> Install packages</a></li>
<li class="chapter" data-level="3.2" data-path="setting-up-geometadb.html"><a href="setting-up-geometadb.html#get-the-sqlite-database"><i class="fa fa-check"></i><b>3.2</b> Get the SQLite database</a></li>
<li class="chapter" data-level="3.3" data-path="setting-up-geometadb.html"><a href="setting-up-geometadb.html#connect-to-the-database"><i class="fa fa-check"></i><b>3.3</b> Connect to the database</a></li>
<li class="chapter" data-level="3.4" data-path="setting-up-geometadb.html"><a href="setting-up-geometadb.html#explore-tables-and-fields"><i class="fa fa-check"></i><b>3.4</b> Explore tables and fields</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="exploring-platforms-gpl.html"><a href="exploring-platforms-gpl.html"><i class="fa fa-check"></i><b>4</b> Exploring Platforms (GPL)</a>
<ul>
<li class="chapter" data-level="4.1" data-path="exploring-platforms-gpl.html"><a href="exploring-platforms-gpl.html#restrict-to-human"><i class="fa fa-check"></i><b>4.1</b> Restrict to human</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="finding-datasets-of-interest.html"><a href="finding-datasets-of-interest.html"><i class="fa fa-check"></i><b>5</b> Finding Datasets of Interest</a>
<ul>
<li class="chapter" data-level="5.1" data-path="finding-datasets-of-interest.html"><a href="finding-datasets-of-interest.html#rna-seq-series-by-platform-technology"><i class="fa fa-check"></i><b>5.1</b> RNA-seq series by platform technology</a></li>
<li class="chapter" data-level="5.2" data-path="finding-datasets-of-interest.html"><a href="finding-datasets-of-interest.html#add-organism-and-recency-filters"><i class="fa fa-check"></i><b>5.2</b> Add organism and recency filters</a></li>
<li class="chapter" data-level="5.3" data-path="finding-datasets-of-interest.html"><a href="finding-datasets-of-interest.html#add-a-topic-keyword-e.g.-ovarian"><i class="fa fa-check"></i><b>5.3</b> Add a topic keyword (e.g., ovarian)</a></li>
<li class="chapter" data-level="5.4" data-path="finding-datasets-of-interest.html"><a href="finding-datasets-of-interest.html#heuristic-require-a-counts-like-supplementary-file"><i class="fa fa-check"></i><b>5.4</b> Heuristic: require a counts-like supplementary file</a></li>
<li class="chapter" data-level="5.5" data-path="finding-datasets-of-interest.html"><a href="finding-datasets-of-interest.html#require-a-minimum-number-of-samples-per-series"><i class="fa fa-check"></i><b>5.5</b> Require a minimum number of samples per series</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="exercises-inspect-specific-geo-series.html"><a href="exercises-inspect-specific-geo-series.html"><i class="fa fa-check"></i><b>6</b> Exercises: Inspect Specific GEO Series</a>
<ul>
<li class="chapter" data-level="6.1" data-path="exercises-inspect-specific-geo-series.html"><a href="exercises-inspect-specific-geo-series.html#exercise-answers"><i class="fa fa-check"></i><b>6.1</b> Exercise Answers:</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="next-week-next-steps.html"><a href="next-week-next-steps.html"><i class="fa fa-check"></i><b>7</b> Next week / Next steps</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Finding Expression Data with GEOmetadb</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="exercises-inspect-specific-geo-series" class="section level1 hasAnchor" number="6">
<h1><span class="header-section-number">Chapter 6</span> Exercises: Inspect Specific GEO Series<a href="exercises-inspect-specific-geo-series.html#exercises-inspect-specific-geo-series" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>In this exercise, you’ll retrieve metadata for the following GEO Series and answer:</p>
<p><strong>GSE IDs:</strong> “GSE201427”,“GSE166530”,“GSE162183”,“GSE186458”</p>
<p><strong>Questions per GSE</strong></p>
<ol style="list-style-type: decimal">
<li><strong>What type of data is it? Describe it.</strong> (platform technology; e.g., high-throughput sequencing, array)</li>
<li><strong>What disease or model is it using?</strong> (infer from series title/summary/platform organism)</li>
<li><strong>How many samples are there? What type of data is associated with the record?</strong></li>
<li><strong>How many control and how many disease?</strong></li>
<li>**Can you find out any information about the GEO dataset - GSE264108. Why or Why not?</li>
</ol>
<blockquote>
<p>Hint: Sample labels are not standardized. Use robust keyword detection (e.g., <code>control|normal|healthy|wild[- ]?type</code> vs. <code>tumou?r|cancer|patient|disease|mutant|treated</code>). Inspect a few GSM rows if the automatic inference is uncertain.</p>
</blockquote>
<div id="exercise-answers" class="section level2 hasAnchor" number="6.1">
<h2><span class="header-section-number">6.1</span> Exercise Answers:<a href="exercises-inspect-specific-geo-series.html#exercise-answers" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div class="sourceCode" id="cb29"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb29-1"><a href="exercises-inspect-specific-geo-series.html#cb29-1" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">requireNamespace</span>(<span class="st">&quot;BiocManager&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb29-2"><a href="exercises-inspect-specific-geo-series.html#cb29-2" tabindex="-1"></a>  <span class="fu">install.packages</span>(<span class="st">&quot;BiocManager&quot;</span>)</span>
<span id="cb29-3"><a href="exercises-inspect-specific-geo-series.html#cb29-3" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">requireNamespace</span>(<span class="st">&quot;GEOmetadb&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb29-4"><a href="exercises-inspect-specific-geo-series.html#cb29-4" tabindex="-1"></a>  BiocManager<span class="sc">::</span><span class="fu">install</span>(<span class="st">&quot;GEOmetadb&quot;</span>)</span>
<span id="cb29-5"><a href="exercises-inspect-specific-geo-series.html#cb29-5" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">requireNamespace</span>(<span class="st">&quot;DBI&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb29-6"><a href="exercises-inspect-specific-geo-series.html#cb29-6" tabindex="-1"></a>  <span class="fu">install.packages</span>(<span class="st">&quot;DBI&quot;</span>)</span>
<span id="cb29-7"><a href="exercises-inspect-specific-geo-series.html#cb29-7" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">requireNamespace</span>(<span class="st">&quot;RSQLite&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb29-8"><a href="exercises-inspect-specific-geo-series.html#cb29-8" tabindex="-1"></a>  <span class="fu">install.packages</span>(<span class="st">&quot;RSQLite&quot;</span>)</span>
<span id="cb29-9"><a href="exercises-inspect-specific-geo-series.html#cb29-9" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">requireNamespace</span>(<span class="st">&quot;dplyr&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb29-10"><a href="exercises-inspect-specific-geo-series.html#cb29-10" tabindex="-1"></a>  <span class="fu">install.packages</span>(<span class="st">&quot;dplyr&quot;</span>)</span>
<span id="cb29-11"><a href="exercises-inspect-specific-geo-series.html#cb29-11" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">requireNamespace</span>(<span class="st">&quot;ggplot2&quot;</span>, <span class="at">quietly =</span> <span class="cn">TRUE</span>))</span>
<span id="cb29-12"><a href="exercises-inspect-specific-geo-series.html#cb29-12" tabindex="-1"></a>  <span class="fu">install.packages</span>(<span class="st">&quot;ggplot2&quot;</span>)</span></code></pre></div>
<div class="sourceCode" id="cb30"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb30-1"><a href="exercises-inspect-specific-geo-series.html#cb30-1" tabindex="-1"></a><span class="fu">library</span>(GEOmetadb)</span>
<span id="cb30-2"><a href="exercises-inspect-specific-geo-series.html#cb30-2" tabindex="-1"></a><span class="cf">if</span> (<span class="sc">!</span><span class="fu">file.exists</span>(<span class="st">&#39;GEOmetadb.sqlite&#39;</span>)) <span class="fu">getSQLiteFile</span>()</span>
<span id="cb30-3"><a href="exercises-inspect-specific-geo-series.html#cb30-3" tabindex="-1"></a><span class="fu">file.info</span>(<span class="st">&#39;GEOmetadb.sqlite&#39;</span>)</span></code></pre></div>
<pre><code>##                         size isdir mode               mtime               ctime               atime  uid  gid   uname  grname
## GEOmetadb.sqlite 19576775680 FALSE  644 2026-01-20 22:58:25 2026-01-21 23:20:53 2026-01-20 22:58:25 1000 1000 rstudio rstudio</code></pre>
<div class="sourceCode" id="cb32"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb32-1"><a href="exercises-inspect-specific-geo-series.html#cb32-1" tabindex="-1"></a><span class="fu">library</span>(DBI)</span>
<span id="cb32-2"><a href="exercises-inspect-specific-geo-series.html#cb32-2" tabindex="-1"></a><span class="fu">library</span>(RSQLite)</span>
<span id="cb32-3"><a href="exercises-inspect-specific-geo-series.html#cb32-3" tabindex="-1"></a>con <span class="ot">&lt;-</span> <span class="fu">dbConnect</span>(<span class="fu">SQLite</span>(), <span class="st">&#39;GEOmetadb.sqlite&#39;</span>)</span></code></pre></div>
<p><strong>GSE201427</strong></p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="exercises-inspect-specific-geo-series.html#cb33-1" tabindex="-1"></a>sql <span class="ot">&lt;-</span> <span class="fu">paste</span>(</span>
<span id="cb33-2"><a href="exercises-inspect-specific-geo-series.html#cb33-2" tabindex="-1"></a>  <span class="st">&quot;SELECT DISTINCT gse.gse, gse.overall_design, gse.type, gsm.type, gse.title, gse.summary, gsm.title, gsm.gpl, gsm.source_name_ch1, gsm.characteristics_ch1, gse.supplementary_file&quot;</span>,</span>
<span id="cb33-3"><a href="exercises-inspect-specific-geo-series.html#cb33-3" tabindex="-1"></a>  <span class="st">&quot;FROM gse JOIN gse_gsm ON gse_gsm.gse = gse.gse&quot;</span>,</span>
<span id="cb33-4"><a href="exercises-inspect-specific-geo-series.html#cb33-4" tabindex="-1"></a>  <span class="st">&quot;JOIN gsm ON gse_gsm.gsm = gsm.gsm&quot;</span>,</span>
<span id="cb33-5"><a href="exercises-inspect-specific-geo-series.html#cb33-5" tabindex="-1"></a>  <span class="st">&quot;WHERE gse_gsm.gse = &#39;GSE201427&#39;&quot;</span></span>
<span id="cb33-6"><a href="exercises-inspect-specific-geo-series.html#cb33-6" tabindex="-1"></a>)</span>
<span id="cb33-7"><a href="exercises-inspect-specific-geo-series.html#cb33-7" tabindex="-1"></a><span class="fu">dbGetQuery</span>(con, sql)</span></code></pre></div>
<pre><code>##         gse
## 1 GSE201427
## 2 GSE201427
## 3 GSE201427
## 4 GSE201427
## 5 GSE201427
## 6 GSE201427
## 7 GSE201427
## 8 GSE201427
##                                                                                                                                                                      overall_design
## 1 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
## 2 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
## 3 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
## 4 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
## 5 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
## 6 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
## 7 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
## 8 Panc1 cells were treated either with control siRNA (siCtrl) or siRNA targeting SF3B1 (siSF3B1). siCtrl cells were exposed to 1% O2 or 21% O2, siSF3B1 cells to 1% O2 for 8 hours.
##                                                 type type                                                                                                     title
## 1 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
## 2 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
## 3 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
## 4 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
## 5 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
## 6 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
## 7 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
## 8 Expression profiling by high throughput sequencing  SRA SF3B1 promotes glycolysis and tumor malignancy through splicing-independent activation of HIF1α [RNA-Seq]
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     summary
## 1 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
## 2 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
## 3 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
## 4 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
## 5 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
## 6 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
## 7 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
## 8 Splicing factor 3b subunit 1 (SF3B1) is one of the most frequently mutated spliceosomal components in cancer. Recent studies additionally imply an involvement of the wildtype protein in tumor progression, but the underlaying mechanism remains elusive.  Here we report that SF3B1 acts as a coactivator for hypoxia-inducible factor (HIF) 1α. SF3B1 directly interacts with HIF1a, augments HIF1a-HIF1b heterodimer binding to hypoxia response elements in vivo and enhances transactivation of HIF target genes. In human patients, primary patient cells and in a mouse model of pancreatic ductal adenocarcinoma (PDAC), efficient HIF1a-signalling relies on SF3B1. Monoallelic deletion of Sf3b1 in this mouse tumor model severely delays tumor initiation and development and diminishes HIF target gene transcription, glucose uptake and proliferation in corresponding tumor organoids in hypoxia, while splicing remains functional. Here, we define a subset of aggressive chromophobe renal cell carcinomas (chRCCs) able to transcriptionally compensate monoallelic SF3B1 loss, a feature that correlates with poor prognosis and lymph node spread and we show that the oncogenic hotspot mutation SF3B1K700E has reduced co-activation efficiency. Also, we report increased sensitivity of cells with low levels of SF3B1 to the HIF-inhibitor PX-478. These findings reveal a splicing-independent, tumor promoting function of SF3B1, important in the pathogenesis of PDAC and other cancers.
##                  title      gpl       source_name_ch1                                        characteristics_ch1
## 1  A-1_Panc1_siCtrl_Nx GPL20301 PDAC cell line (ATCC) condition: Normoxia;\ttreatment: siCtrl;\tcell line: Panc1
## 2  A-4_Panc1_siCtrl_Nx GPL20301 PDAC cell line (ATCC) condition: Normoxia;\ttreatment: siCtrl;\tcell line: Panc1
## 3  A-2_Panc1_siCtrl_Hx GPL20301 PDAC cell line (ATCC)  condition: Hypoxia;\ttreatment: siCtrl;\tcell line: Panc1
## 4  A-5_Panc1_siCtrl_Hx GPL20301 PDAC cell line (ATCC)  condition: Hypoxia;\ttreatment: siCtrl;\tcell line: Panc1
## 5  A-8_Panc1_siCtrl_Hx GPL20301 PDAC cell line (ATCC)  condition: Hypoxia;\ttreatment: siCtrl;\tcell line: Panc1
## 6 A-3_Panc1_siSF3B1_Hx GPL20301 PDAC cell line (ATCC) condition: Hypoxia;\ttreatment: siSF3B1;\tcell line: Panc1
## 7 A-6_Panc1_siSF3B1_Hx GPL20301 PDAC cell line (ATCC) condition: Hypoxia;\ttreatment: siSF3B1;\tcell line: Panc1
## 8 A-9_Panc1_siSF3B1_Hx GPL20301 PDAC cell line (ATCC) condition: Hypoxia;\ttreatment: siSF3B1;\tcell line: Panc1
##                                                                                 supplementary_file
## 1 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx
## 2 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx
## 3 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx
## 4 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx
## 5 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx
## 6 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx
## 7 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx
## 8 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE201nnn/GSE201427/suppl/GSE201427_CPM_RNA-Seq_Panc1.xlsx</code></pre>
<ol style="list-style-type: decimal">
<li>The GSE201427 study uses high throughoutput sequencing for expression profilling.</li>
<li>A cell line for cancer is used where the effect of a siRNA is investigated for knock-down of SF3B1 splicing factor.</li>
<li>There are 8 samples used in the study and the associated data is RNA-seq with provided counts per million files.</li>
<li>Within the 8 samples 5 correspond to control and 3 are the siSF3B1 positive group.</li>
</ol>
<p><strong>GSE166530</strong></p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="exercises-inspect-specific-geo-series.html#cb35-1" tabindex="-1"></a>sql <span class="ot">&lt;-</span> <span class="fu">paste</span>(</span>
<span id="cb35-2"><a href="exercises-inspect-specific-geo-series.html#cb35-2" tabindex="-1"></a>  <span class="st">&quot;SELECT DISTINCT gse.gse, gse.overall_design, gse.type, gsm.type, gse.title, gse.summary, gsm.title, gsm.gpl, gsm.source_name_ch1, gsm.characteristics_ch1, gse.supplementary_file&quot;</span>,</span>
<span id="cb35-3"><a href="exercises-inspect-specific-geo-series.html#cb35-3" tabindex="-1"></a>  <span class="st">&quot;FROM gse JOIN gse_gsm ON gse_gsm.gse = gse.gse&quot;</span>,</span>
<span id="cb35-4"><a href="exercises-inspect-specific-geo-series.html#cb35-4" tabindex="-1"></a>  <span class="st">&quot;JOIN gsm ON gse_gsm.gsm = gsm.gsm&quot;</span>,</span>
<span id="cb35-5"><a href="exercises-inspect-specific-geo-series.html#cb35-5" tabindex="-1"></a>  <span class="st">&quot;WHERE gse_gsm.gse = &#39;GSE166530&#39;&quot;</span></span>
<span id="cb35-6"><a href="exercises-inspect-specific-geo-series.html#cb35-6" tabindex="-1"></a>)</span>
<span id="cb35-7"><a href="exercises-inspect-specific-geo-series.html#cb35-7" tabindex="-1"></a><span class="fu">dbGetQuery</span>(con, sql)</span></code></pre></div>
<pre><code>##          gse                                                                                                    overall_design                                               type type
## 1  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 2  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 3  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 4  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 5  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 6  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 7  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 8  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 9  GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 10 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 11 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 12 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 13 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 14 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 15 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 16 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 17 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 18 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 19 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 20 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 21 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 22 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 23 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 24 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 25 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 26 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 27 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 28 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 29 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 30 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 31 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 32 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 33 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 34 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 35 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 36 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 37 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 38 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 39 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 40 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
## 41 GSE166530 Differential expression analysis of RNA-Seq data from COVID-19 positive samples against COVID-19 negative control Expression profiling by high throughput sequencing  SRA
##                                                                    title
## 1  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 2  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 3  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 4  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 5  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 6  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 7  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 8  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 9  Deciphering the host transcriptional response to SARS-CoV-2 infection
## 10 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 11 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 12 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 13 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 14 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 15 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 16 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 17 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 18 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 19 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 20 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 21 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 22 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 23 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 24 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 25 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 26 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 27 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 28 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 29 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 30 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 31 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 32 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 33 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 34 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 35 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 36 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 37 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 38 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 39 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 40 Deciphering the host transcriptional response to SARS-CoV-2 infection
## 41 Deciphering the host transcriptional response to SARS-CoV-2 infection
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     summary
## 1  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 2  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 3  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 4  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 5  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 6  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 7  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 8  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 9  SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 10 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 11 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 12 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 13 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 14 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 15 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 16 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 17 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 18 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 19 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 20 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 21 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 22 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 23 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 24 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 25 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 26 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 27 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 28 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 29 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 30 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 31 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 32 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 33 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 34 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 35 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 36 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 37 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 38 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 39 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 40 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
## 41 SARS-CoV-2 is a beta coronavirus causing COVID-19 which first emerged in Wuhan, China and was later declared a pandemic by the World Health Organization. Since then the economical, health and human cost has been enormous for the world. However, little work has been done to understand the transcriptional changes brought about by the virus in human hosts. We have compared COVID-19 positive samples with negative samples from Indian patients to better understand the host response.. We find many genes related to immune response  up-regulated in the COVID-19 patients. Many of these are the usual response genes against the viral infection but type I interferon appears to be a  key immune response activated against SARS-CoV-2. A large number of the differentially expressed genes were down-regulated pointing towards translational arrest and down regulation of host mRNA during late infection. The down-regulated genes are well correlated with the clinical manifestations and symptoms due to SARS-CoV-2 infection such as the loss of smell and taste. We also find evidence of altered gene expression profiles associated with systemic complications such as neurological disturbances and high insulin requirement. Finally, we have identified many lncRNAs being down-regulated during COVID-19 infections. A few of these lncRNAs have functional role in viral infection. However, to understand the functional role of other lncRNAs, we looked at the function of their closest gene, since lncRNA are believed to have cis functionality. Our analysis suggests a role for lncRNA in down-regulation of metabolic and developmental processes during COVID-19 infection.
##                               title      gpl            source_name_ch1
## 1   COVID-19 Negative samples rep 1 GPL24676 COVID-19 negative controls
## 2   COVID-19 Negative samples rep 2 GPL24676 COVID-19 negative controls
## 3   COVID-19 Negative samples rep 3 GPL24676 COVID-19 negative controls
## 4   COVID-19 Negative samples rep 4 GPL24676 COVID-19 negative controls
## 5   COVID-19 Negative samples rep 5 GPL24676 COVID-19 negative controls
## 6   COVID-19 Positive samples rep 1 GPL24676  COVID-19 positive samples
## 7   COVID-19 Positive samples rep 2 GPL24676  COVID-19 positive samples
## 8   COVID-19 Positive samples rep 3 GPL24676  COVID-19 positive samples
## 9   COVID-19 Positive samples rep 4 GPL24676  COVID-19 positive samples
## 10  COVID-19 Positive samples rep 5 GPL24676  COVID-19 positive samples
## 11  COVID-19 Positive samples rep 6 GPL24676  COVID-19 positive samples
## 12  COVID-19 Positive samples rep 7 GPL24676  COVID-19 positive samples
## 13  COVID-19 Positive samples rep 8 GPL24676  COVID-19 positive samples
## 14  COVID-19 Positive samples rep 9 GPL24676  COVID-19 positive samples
## 15 COVID-19 Positive samples rep 10 GPL24676  COVID-19 positive samples
## 16 COVID-19 Positive samples rep 11 GPL24676  COVID-19 positive samples
## 17 COVID-19 Positive samples rep 12 GPL24676  COVID-19 positive samples
## 18 COVID-19 Positive samples rep 13 GPL24676  COVID-19 positive samples
## 19 COVID-19 Positive samples rep 14 GPL24676  COVID-19 positive samples
## 20 COVID-19 Positive samples rep 15 GPL24676  COVID-19 positive samples
## 21 COVID-19 Positive samples rep 16 GPL24676  COVID-19 positive samples
## 22 COVID-19 Positive samples rep 17 GPL24676  COVID-19 positive samples
## 23 COVID-19 Positive samples rep 18 GPL24676  COVID-19 positive samples
## 24 COVID-19 Positive samples rep 19 GPL24676  COVID-19 positive samples
## 25 COVID-19 Positive samples rep 20 GPL24676  COVID-19 positive samples
## 26 COVID-19 Positive samples rep 21 GPL24676  COVID-19 positive samples
## 27 COVID-19 Positive samples rep 22 GPL24676  COVID-19 positive samples
## 28 COVID-19 Positive samples rep 23 GPL24676  COVID-19 positive samples
## 29 COVID-19 Positive samples rep 24 GPL24676  COVID-19 positive samples
## 30 COVID-19 Positive samples rep 25 GPL24676  COVID-19 positive samples
## 31 COVID-19 Positive samples rep 26 GPL24676  COVID-19 positive samples
## 32 COVID-19 Positive samples rep 27 GPL24676  COVID-19 positive samples
## 33 COVID-19 Positive samples rep 28 GPL24676  COVID-19 positive samples
## 34 COVID-19 Positive samples rep 29 GPL24676  COVID-19 positive samples
## 35 COVID-19 Positive samples rep 30 GPL24676  COVID-19 positive samples
## 36 COVID-19 Positive samples rep 31 GPL24676  COVID-19 positive samples
## 37 COVID-19 Positive samples rep 32 GPL24676  COVID-19 positive samples
## 38 COVID-19 Positive samples rep 33 GPL24676  COVID-19 positive samples
## 39 COVID-19 Positive samples rep 34 GPL24676  COVID-19 positive samples
## 40 COVID-19 Positive samples rep 35 GPL24676  COVID-19 positive samples
## 41 COVID-19 Positive samples rep 36 GPL24676  COVID-19 positive samples
##                                                                                                            characteristics_ch1
## 1  subject status: COVID-19 Negative;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 2  subject status: COVID-19 Negative;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 3  subject status: COVID-19 Negative;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 4  subject status: COVID-19 Negative;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 5  subject status: COVID-19 Negative;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 6  subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 7  subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 8  subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 9  subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 10 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 11 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 12 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 13 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 14 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 15 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 16 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 17 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 18 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 19 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 20 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 21 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 22 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 23 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 24 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 25 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 26 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 27 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 28 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 29 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 30 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 31 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 32 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 33 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 34 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 35 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 36 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 37 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 38 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 39 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 40 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
## 41 subject status: COVID-19 Positive;\tsample type: nasopharyngeal or oropharyngeal swabs;\tmolecule subtype: ribodepleted RNA
##                                                                   supplementary_file
## 1  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 2  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 3  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 4  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 5  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 6  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 7  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 8  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 9  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 10 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 11 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 12 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 13 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 14 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 15 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 16 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 17 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 18 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 19 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 20 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 21 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 22 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 23 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 24 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 25 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 26 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 27 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 28 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 29 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 30 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 31 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 32 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 33 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 34 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 35 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 36 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 37 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 38 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 39 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 40 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar
## 41 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE166nnn/GSE166530/suppl/GSE166530_RAW.tar</code></pre>
<ol style="list-style-type: decimal">
<li>The GSE166530 study is high throughput sequencing of RNA expression profiling.</li>
<li>It is using a disease model for COVID-19 sampled from humans.</li>
<li>There are 10 samples included in the study. Each sample has associated RNA-seq data.</li>
<li>The 10 samples are composed of an equal split of 5 control and 5 positive for COVID-19.</li>
</ol>
<p><strong>GSE162183</strong></p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="exercises-inspect-specific-geo-series.html#cb37-1" tabindex="-1"></a>sql <span class="ot">&lt;-</span> <span class="fu">paste</span>(</span>
<span id="cb37-2"><a href="exercises-inspect-specific-geo-series.html#cb37-2" tabindex="-1"></a>  <span class="st">&quot;SELECT DISTINCT gse.gse, gse.overall_design, gse.type, gsm.type, gse.title, gse.summary, gsm.title, gsm.gpl, gsm.source_name_ch1, gsm.characteristics_ch1, gse.supplementary_file&quot;</span>,</span>
<span id="cb37-3"><a href="exercises-inspect-specific-geo-series.html#cb37-3" tabindex="-1"></a>  <span class="st">&quot;FROM gse JOIN gse_gsm ON gse_gsm.gse = gse.gse&quot;</span>,</span>
<span id="cb37-4"><a href="exercises-inspect-specific-geo-series.html#cb37-4" tabindex="-1"></a>  <span class="st">&quot;JOIN gsm ON gse_gsm.gsm = gsm.gsm&quot;</span>,</span>
<span id="cb37-5"><a href="exercises-inspect-specific-geo-series.html#cb37-5" tabindex="-1"></a>  <span class="st">&quot;WHERE gse_gsm.gse = &#39;GSE162183&#39;&quot;</span></span>
<span id="cb37-6"><a href="exercises-inspect-specific-geo-series.html#cb37-6" tabindex="-1"></a>)</span>
<span id="cb37-7"><a href="exercises-inspect-specific-geo-series.html#cb37-7" tabindex="-1"></a><span class="fu">dbGetQuery</span>(con, sql)</span></code></pre></div>
<pre><code>##         gse
## 1 GSE162183
## 2 GSE162183
## 3 GSE162183
## 4 GSE162183
## 5 GSE162183
## 6 GSE162183
##                                                                                                                                                                                           overall_design
## 1 Single cell RNA-seq study of human psoriasis skin;\t;\t*** Submitter declares that the raw data will be deposited in the European Genome-phenome Archive (EGA) due to patient privacy concerns. ***;\t
## 2 Single cell RNA-seq study of human psoriasis skin;\t;\t*** Submitter declares that the raw data will be deposited in the European Genome-phenome Archive (EGA) due to patient privacy concerns. ***;\t
## 3 Single cell RNA-seq study of human psoriasis skin;\t;\t*** Submitter declares that the raw data will be deposited in the European Genome-phenome Archive (EGA) due to patient privacy concerns. ***;\t
## 4 Single cell RNA-seq study of human psoriasis skin;\t;\t*** Submitter declares that the raw data will be deposited in the European Genome-phenome Archive (EGA) due to patient privacy concerns. ***;\t
## 5 Single cell RNA-seq study of human psoriasis skin;\t;\t*** Submitter declares that the raw data will be deposited in the European Genome-phenome Archive (EGA) due to patient privacy concerns. ***;\t
## 6 Single cell RNA-seq study of human psoriasis skin;\t;\t*** Submitter declares that the raw data will be deposited in the European Genome-phenome Archive (EGA) due to patient privacy concerns. ***;\t
##                                                 type type                                                              title
## 1 Expression profiling by high throughput sequencing  SRA Single cell transcriptional zonation of human psoriasis whole skin
## 2 Expression profiling by high throughput sequencing  SRA Single cell transcriptional zonation of human psoriasis whole skin
## 3 Expression profiling by high throughput sequencing  SRA Single cell transcriptional zonation of human psoriasis whole skin
## 4 Expression profiling by high throughput sequencing  SRA Single cell transcriptional zonation of human psoriasis whole skin
## 5 Expression profiling by high throughput sequencing  SRA Single cell transcriptional zonation of human psoriasis whole skin
## 6 Expression profiling by high throughput sequencing  SRA Single cell transcriptional zonation of human psoriasis whole skin
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             summary
## 1 To explore the cellullar and molecular alteration of human psoriasis, we collected full-thickness skin from the lesion region of 3 patients and the similar region of 3 healthy donors, and submit for single cell RNA sequencing (scRNAseq) with 10x genomics (V3.1). The transcriptional landscape of human psoriasis whole skin provide a unique view of immuno-regulation among skin cell types.;\t;\t&quot;Single cell transcriptional zonation of human psoriasis skin identifies an alternative immunoregulatory axis&quot;,&lt;Cell Death Dis.&gt;, 2021 May 6;12(5):450.
## 2 To explore the cellullar and molecular alteration of human psoriasis, we collected full-thickness skin from the lesion region of 3 patients and the similar region of 3 healthy donors, and submit for single cell RNA sequencing (scRNAseq) with 10x genomics (V3.1). The transcriptional landscape of human psoriasis whole skin provide a unique view of immuno-regulation among skin cell types.;\t;\t&quot;Single cell transcriptional zonation of human psoriasis skin identifies an alternative immunoregulatory axis&quot;,&lt;Cell Death Dis.&gt;, 2021 May 6;12(5):450.
## 3 To explore the cellullar and molecular alteration of human psoriasis, we collected full-thickness skin from the lesion region of 3 patients and the similar region of 3 healthy donors, and submit for single cell RNA sequencing (scRNAseq) with 10x genomics (V3.1). The transcriptional landscape of human psoriasis whole skin provide a unique view of immuno-regulation among skin cell types.;\t;\t&quot;Single cell transcriptional zonation of human psoriasis skin identifies an alternative immunoregulatory axis&quot;,&lt;Cell Death Dis.&gt;, 2021 May 6;12(5):450.
## 4 To explore the cellullar and molecular alteration of human psoriasis, we collected full-thickness skin from the lesion region of 3 patients and the similar region of 3 healthy donors, and submit for single cell RNA sequencing (scRNAseq) with 10x genomics (V3.1). The transcriptional landscape of human psoriasis whole skin provide a unique view of immuno-regulation among skin cell types.;\t;\t&quot;Single cell transcriptional zonation of human psoriasis skin identifies an alternative immunoregulatory axis&quot;,&lt;Cell Death Dis.&gt;, 2021 May 6;12(5):450.
## 5 To explore the cellullar and molecular alteration of human psoriasis, we collected full-thickness skin from the lesion region of 3 patients and the similar region of 3 healthy donors, and submit for single cell RNA sequencing (scRNAseq) with 10x genomics (V3.1). The transcriptional landscape of human psoriasis whole skin provide a unique view of immuno-regulation among skin cell types.;\t;\t&quot;Single cell transcriptional zonation of human psoriasis skin identifies an alternative immunoregulatory axis&quot;,&lt;Cell Death Dis.&gt;, 2021 May 6;12(5):450.
## 6 To explore the cellullar and molecular alteration of human psoriasis, we collected full-thickness skin from the lesion region of 3 patients and the similar region of 3 healthy donors, and submit for single cell RNA sequencing (scRNAseq) with 10x genomics (V3.1). The transcriptional landscape of human psoriasis whole skin provide a unique view of immuno-regulation among skin cell types.;\t;\t&quot;Single cell transcriptional zonation of human psoriasis skin identifies an alternative immunoregulatory axis&quot;,&lt;Cell Death Dis.&gt;, 2021 May 6;12(5):450.
##                         title      gpl source_name_ch1                     characteristics_ch1
## 1 skin control1 10x scRNA-seq GPL24676        control1   tissue: skin;\tdisease state: healthy
## 2 skin control2 10x scRNA-seq GPL24676        control2   tissue: skin;\tdisease state: healthy
## 3 skin control3 10x scRNA-seq GPL24676        control3   tissue: skin;\tdisease state: healthy
## 4    psoriasis1 10x scRNA-seq GPL24676      psoriasis1 tissue: skin;\tdisease state: psoriasis
## 5    psoriasis2 10x scRNA-seq GPL24676      psoriasis2 tissue: skin;\tdisease state: psoriasis
## 6    psoriasis3 10x scRNA-seq GPL24676      psoriasis3 tissue: skin;\tdisease state: psoriasis
##                                                                                                                                                                                                            supplementary_file
## 1 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix.tab.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix_LoomFile.loom.gz
## 2 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix.tab.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix_LoomFile.loom.gz
## 3 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix.tab.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix_LoomFile.loom.gz
## 4 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix.tab.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix_LoomFile.loom.gz
## 5 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix.tab.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix_LoomFile.loom.gz
## 6 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix.tab.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE162nnn/GSE162183/suppl/GSE162183_Raw_gene_counts_matrix_LoomFile.loom.gz</code></pre>
<ol style="list-style-type: decimal">
<li>The GSE162183 study is high throughput sequencing of single cell RNA-seq.</li>
<li>It is using tissue samples from psoriasis and control donors as a model for the psoriasis disease.</li>
<li>There are 6 samples where each sample undergoes 10 replicates of scRNA-seq.</li>
<li>The samples are half control and half positive with 3 corresponding to either group. Each has associated single cell RNA sequencing data with provided the provided gene expression matrices.</li>
</ol>
<p><strong>GSE186458</strong></p>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="exercises-inspect-specific-geo-series.html#cb39-1" tabindex="-1"></a>sql <span class="ot">&lt;-</span> <span class="fu">paste</span>(</span>
<span id="cb39-2"><a href="exercises-inspect-specific-geo-series.html#cb39-2" tabindex="-1"></a>  <span class="st">&quot;SELECT DISTINCT gse.gse, gse.overall_design, gse.type, gsm.type, gse.title, gse.summary, gsm.title, gsm.gpl, gsm.source_name_ch1, gsm.characteristics_ch1, gse.supplementary_file&quot;</span>,</span>
<span id="cb39-3"><a href="exercises-inspect-specific-geo-series.html#cb39-3" tabindex="-1"></a>  <span class="st">&quot;FROM gse JOIN gse_gsm ON gse_gsm.gse = gse.gse&quot;</span>,</span>
<span id="cb39-4"><a href="exercises-inspect-specific-geo-series.html#cb39-4" tabindex="-1"></a>  <span class="st">&quot;JOIN gsm ON gse_gsm.gsm = gsm.gsm&quot;</span>,</span>
<span id="cb39-5"><a href="exercises-inspect-specific-geo-series.html#cb39-5" tabindex="-1"></a>  <span class="st">&quot;WHERE gse_gsm.gse = &#39;GSE186458&#39;&quot;</span></span>
<span id="cb39-6"><a href="exercises-inspect-specific-geo-series.html#cb39-6" tabindex="-1"></a>)</span>
<span id="cb39-7"><a href="exercises-inspect-specific-geo-series.html#cb39-7" tabindex="-1"></a><span class="fu">dbGetQuery</span>(con, sql)</span></code></pre></div>
<pre><code>##          gse
## 1  GSE186458
## 2  GSE186458
## 3  GSE186458
## 4  GSE186458
## 5  GSE186458
## 6  GSE186458
## 7  GSE186458
## 8  GSE186458
## 9  GSE186458
## 10 GSE186458
## 11 GSE186458
## 12 GSE186458
## 13 GSE186458
## 14 GSE186458
## 15 GSE186458
## 16 GSE186458
## 17 GSE186458
## 18 GSE186458
## 19 GSE186458
## 20 GSE186458
## 21 GSE186458
## 22 GSE186458
## 23 GSE186458
## 24 GSE186458
## 25 GSE186458
## 26 GSE186458
## 27 GSE186458
## 28 GSE186458
## 29 GSE186458
## 30 GSE186458
## 31 GSE186458
## 32 GSE186458
## 33 GSE186458
## 34 GSE186458
## 35 GSE186458
## 36 GSE186458
## 37 GSE186458
## 38 GSE186458
## 39 GSE186458
## 40 GSE186458
## 41 GSE186458
## 42 GSE186458
## 43 GSE186458
## 44 GSE186458
## 45 GSE186458
## 46 GSE186458
## 47 GSE186458
## 48 GSE186458
## 49 GSE186458
## 50 GSE186458
## 51 GSE186458
## 52 GSE186458
## 53 GSE186458
## 54 GSE186458
## 55 GSE186458
## 56 GSE186458
## 57 GSE186458
## 58 GSE186458
## 59 GSE186458
## 60 GSE186458
## 61 GSE186458
## 62 GSE186458
## 63 GSE186458
## 64 GSE186458
## 65 GSE186458
## 66 GSE186458
## 67 GSE186458
## 68 GSE186458
## 69 GSE186458
## 70 GSE186458
## 71 GSE186458
## 72 GSE186458
## 73 GSE186458
## 74 GSE186458
## 75 GSE186458
## 76 GSE186458
## 77 GSE186458
## 78 GSE186458
## 79 GSE186458
## 80 GSE186458
## 81 GSE186458
## 82 GSE186458
## 83 GSE186458
## 84 GSE186458
## 85 GSE186458
## 86 GSE186458
## 87 GSE186458
## 88 GSE186458
## 89 GSE186458
## 90 GSE186458
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   overall_design
## 1  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 2  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 3  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 4  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 5  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 6  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 7  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 8  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 9  To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 10 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 11 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 12 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 13 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 14 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 15 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 16 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 17 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 18 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 19 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 20 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 21 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 22 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 23 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 24 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 25 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 26 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 27 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 28 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 29 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 30 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 31 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 32 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 33 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 34 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 35 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 36 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 37 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 38 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 39 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 40 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 41 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 42 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 43 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 44 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 45 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 46 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 47 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 48 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 49 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 50 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 51 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 52 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 53 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 54 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 55 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 56 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 57 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 58 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 59 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 60 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 61 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 62 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 63 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 64 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 65 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 66 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 67 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 68 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 69 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 70 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 71 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 72 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 73 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 74 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 75 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 76 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 77 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 78 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 79 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 80 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 81 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 82 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 83 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 84 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 85 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 86 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 87 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 88 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 89 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
## 90 To portray the genome-wide patterns of DNA methylation across a variety of cell types, we obtained 205 samples of freshly isolated healthy adult tissue samples from 137 consented donors undergoing a variety of surgical procedures (ages 3-83). We dissociated tissue samples into single cell suspensions, and used lineage-specific antibodies to cell type specific surface markers to FACS purify cell populations covering 39 primary cell-types. The purity of cell types was confirmed using RT-qPCR for cell type specific gene expression markers, and assessing known tissue-specific methylation markers when possible (Methods, Supplemental Fig. S1). We then subjected cell type-specific genomic DNA to WGBS and sequenced at a mean depth of &gt;30X, using 150bp-long paired-end reads, with an average fragment size of 174bp.;\t;\t&gt;&gt;&gt;Submitter states that raw data are not available due to patient privacy concerns&lt;&lt;&lt;;\t;\tPlease note that the records have been updated with the processed data generated with the *hg38* reference on May 17, 2022.
##                                                   type type
## 1  Methylation profiling by high throughput sequencing  SRA
## 2  Methylation profiling by high throughput sequencing  SRA
## 3  Methylation profiling by high throughput sequencing  SRA
## 4  Methylation profiling by high throughput sequencing  SRA
## 5  Methylation profiling by high throughput sequencing  SRA
## 6  Methylation profiling by high throughput sequencing  SRA
## 7  Methylation profiling by high throughput sequencing  SRA
## 8  Methylation profiling by high throughput sequencing  SRA
## 9  Methylation profiling by high throughput sequencing  SRA
## 10 Methylation profiling by high throughput sequencing  SRA
## 11 Methylation profiling by high throughput sequencing  SRA
## 12 Methylation profiling by high throughput sequencing  SRA
## 13 Methylation profiling by high throughput sequencing  SRA
## 14 Methylation profiling by high throughput sequencing  SRA
## 15 Methylation profiling by high throughput sequencing  SRA
## 16 Methylation profiling by high throughput sequencing  SRA
## 17 Methylation profiling by high throughput sequencing  SRA
## 18 Methylation profiling by high throughput sequencing  SRA
## 19 Methylation profiling by high throughput sequencing  SRA
## 20 Methylation profiling by high throughput sequencing  SRA
## 21 Methylation profiling by high throughput sequencing  SRA
## 22 Methylation profiling by high throughput sequencing  SRA
## 23 Methylation profiling by high throughput sequencing  SRA
## 24 Methylation profiling by high throughput sequencing  SRA
## 25 Methylation profiling by high throughput sequencing  SRA
## 26 Methylation profiling by high throughput sequencing  SRA
## 27 Methylation profiling by high throughput sequencing  SRA
## 28 Methylation profiling by high throughput sequencing  SRA
## 29 Methylation profiling by high throughput sequencing  SRA
## 30 Methylation profiling by high throughput sequencing  SRA
## 31 Methylation profiling by high throughput sequencing  SRA
## 32 Methylation profiling by high throughput sequencing  SRA
## 33 Methylation profiling by high throughput sequencing  SRA
## 34 Methylation profiling by high throughput sequencing  SRA
## 35 Methylation profiling by high throughput sequencing  SRA
## 36 Methylation profiling by high throughput sequencing  SRA
## 37 Methylation profiling by high throughput sequencing  SRA
## 38 Methylation profiling by high throughput sequencing  SRA
## 39 Methylation profiling by high throughput sequencing  SRA
## 40 Methylation profiling by high throughput sequencing  SRA
## 41 Methylation profiling by high throughput sequencing  SRA
## 42 Methylation profiling by high throughput sequencing  SRA
## 43 Methylation profiling by high throughput sequencing  SRA
## 44 Methylation profiling by high throughput sequencing  SRA
## 45 Methylation profiling by high throughput sequencing  SRA
## 46 Methylation profiling by high throughput sequencing  SRA
## 47 Methylation profiling by high throughput sequencing  SRA
## 48 Methylation profiling by high throughput sequencing  SRA
## 49 Methylation profiling by high throughput sequencing  SRA
## 50 Methylation profiling by high throughput sequencing  SRA
## 51 Methylation profiling by high throughput sequencing  SRA
## 52 Methylation profiling by high throughput sequencing  SRA
## 53 Methylation profiling by high throughput sequencing  SRA
## 54 Methylation profiling by high throughput sequencing  SRA
## 55 Methylation profiling by high throughput sequencing  SRA
## 56 Methylation profiling by high throughput sequencing  SRA
## 57 Methylation profiling by high throughput sequencing  SRA
## 58 Methylation profiling by high throughput sequencing  SRA
## 59 Methylation profiling by high throughput sequencing  SRA
## 60 Methylation profiling by high throughput sequencing  SRA
## 61 Methylation profiling by high throughput sequencing  SRA
## 62 Methylation profiling by high throughput sequencing  SRA
## 63 Methylation profiling by high throughput sequencing  SRA
## 64 Methylation profiling by high throughput sequencing  SRA
## 65 Methylation profiling by high throughput sequencing  SRA
## 66 Methylation profiling by high throughput sequencing  SRA
## 67 Methylation profiling by high throughput sequencing  SRA
## 68 Methylation profiling by high throughput sequencing  SRA
## 69 Methylation profiling by high throughput sequencing  SRA
## 70 Methylation profiling by high throughput sequencing  SRA
## 71 Methylation profiling by high throughput sequencing  SRA
## 72 Methylation profiling by high throughput sequencing  SRA
## 73 Methylation profiling by high throughput sequencing  SRA
## 74 Methylation profiling by high throughput sequencing  SRA
## 75 Methylation profiling by high throughput sequencing  SRA
## 76 Methylation profiling by high throughput sequencing  SRA
## 77 Methylation profiling by high throughput sequencing  SRA
## 78 Methylation profiling by high throughput sequencing  SRA
## 79 Methylation profiling by high throughput sequencing  SRA
## 80 Methylation profiling by high throughput sequencing  SRA
## 81 Methylation profiling by high throughput sequencing  SRA
## 82 Methylation profiling by high throughput sequencing  SRA
## 83 Methylation profiling by high throughput sequencing  SRA
## 84 Methylation profiling by high throughput sequencing  SRA
## 85 Methylation profiling by high throughput sequencing  SRA
## 86 Methylation profiling by high throughput sequencing  SRA
## 87 Methylation profiling by high throughput sequencing  SRA
## 88 Methylation profiling by high throughput sequencing  SRA
## 89 Methylation profiling by high throughput sequencing  SRA
## 90 Methylation profiling by high throughput sequencing  SRA
##                                                                                                                                                    title
## 1  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 2  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 3  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 4  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 5  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 6  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 7  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 8  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 9  A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 10 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 11 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 12 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 13 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 14 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 15 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 16 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 17 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 18 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 19 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 20 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 21 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 22 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 23 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 24 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 25 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 26 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 27 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 28 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 29 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 30 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 31 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 32 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 33 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 34 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 35 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 36 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 37 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 38 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 39 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 40 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 41 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 42 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 43 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 44 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 45 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 46 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 47 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 48 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 49 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 50 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 51 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 52 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 53 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 54 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 55 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 56 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 57 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 58 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 59 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 60 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 61 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 62 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 63 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 64 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 65 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 66 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 67 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 68 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 69 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 70 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 71 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 72 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 73 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 74 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 75 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 76 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 77 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 78 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 79 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 80 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 81 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 82 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 83 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 84 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 85 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 86 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 87 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 88 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 89 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
## 90 A human DNA methylation atlas reveals principles of cell type-specific methylation and identifies thousands of cell type-specific regulatory elements
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  summary
## 1  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 2  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 3  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 4  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 5  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 6  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 7  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 8  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 9  DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 10 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 11 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 12 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 13 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 14 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 15 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 16 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 17 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 18 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 19 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 20 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 21 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 22 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 23 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 24 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 25 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 26 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 27 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 28 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 29 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 30 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 31 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 32 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 33 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 34 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 35 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 36 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 37 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 38 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 39 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 40 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 41 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 42 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 43 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 44 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 45 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 46 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 47 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 48 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 49 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 50 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 51 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 52 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 53 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 54 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 55 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 56 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 57 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 58 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 59 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 60 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 61 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 62 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 63 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 64 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 65 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 66 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 67 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 68 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 69 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 70 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 71 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 72 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 73 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 74 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 75 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 76 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 77 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 78 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 79 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 80 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 81 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 82 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 83 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 84 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 85 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 86 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 87 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 88 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 89 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
## 90 DNA methylation is a fundamental epigenetic mark that governs chromatin organization, cell identity, and gene expression. Here we describe a human methylome atlas, based on deep whole-genome bisulfite sequencing of 39 cell types sorted from 205 healthy tissue samples.  Replicates of the same cell type are &gt;99.5% identical, demonstrating robustness of cell identity programs to genetic variation and environmental perturbation. Unsupervised clustering of the atlas recapitulates key elements of tissue ontogeny, and identifies methylation patterns retained since gastrulation. Loci uniquely unmethylated in an individual cell type often reside in transcriptional enhancers and contain DNA binding sites for tissue-specific transcriptional regulators. Uniquely methylated loci are rare and are enriched for CpG islands, polycomb targets, and CTCF binding sites, suggesting a role in shaping cell type-specific chromatin looping. The atlas provides an essential resource for interpretation of disease-associated genetic variants, and a wealth of potential tissue-specific biomarkers for use in liquid biopsies.
##                                      title      gpl    source_name_ch1                                                                                 characteristics_ch1
## 1                     Adipocytes-Z000000T7 GPL24676  Abdominal Subcut.           tissue: Abdominal Subcut.;\tcell type: Adipocytes;\tlab: KI;\tage: 53;\tSex: F;\trace: NA
## 2                     Adipocytes-Z000000T9 GPL24676  Abdominal Subcut.           tissue: Abdominal Subcut.;\tcell type: Adipocytes;\tlab: KI;\tage: 35;\tSex: F;\trace: NA
## 3                     Adipocytes-Z000000T5 GPL24676  Abdominal Subcut.           tissue: Abdominal Subcut.;\tcell type: Adipocytes;\tlab: KI;\tage: 37;\tSex: F;\trace: NA
## 4              Aorta-Endothelium-Z00000422 GPL24676              Aorta                tissue: Aorta;\tcell type: Endothelium;\tlab: Lonza;\tage: 33;\tSex: F;\trace: White
## 5              Aorta-Endothelium-Z0000043G GPL24676              Aorta               tissue: Aorta;\tcell type: Endothelium;\tlab: Had-HU;\tage: 62;\tSex: F;\trace: White
## 6     Saphenous-Vein-Endothelium-Z000000RM GPL24676 Vascular saphenous  tissue: Vascular saphenous;\tcell type: Endothelium;\tlab: Had-HU;\tage: 54;\tSex: F;\trace: White
## 7     Saphenous-Vein-Endothelium-Z000000S7 GPL24676 Vascular saphenous  tissue: Vascular saphenous;\tcell type: Endothelium;\tlab: Had-HU;\tage: 52;\tSex: F;\trace: White
## 8     Saphenous-Vein-Endothelium-Z000000SB GPL24676 Vascular saphenous  tissue: Vascular saphenous;\tcell type: Endothelium;\tlab: Had-HU;\tage: 34;\tSex: M;\trace: White
## 9  Kidney-Glomerular-Endothelium-Z000000Q5 GPL24676  Kidney glomerular   tissue: Kidney glomerular;\tcell type: Endothelium;\tlab: Had-HU;\tage: 67;\tSex: M;\trace: White
## 10 Kidney-Glomerular-Endothelium-Z00000443 GPL24676  Kidney glomerular   tissue: Kidney glomerular;\tcell type: Endothelium;\tlab: Had-HU;\tage: 60;\tSex: M;\trace: White
## 11 Kidney-Glomerular-Endothelium-Z0000045J GPL24676  Kidney glomerular   tissue: Kidney glomerular;\tcell type: Endothelium;\tlab: Had-HU;\tage: 47;\tSex: M;\trace: White
## 12    Kidney-Tubular-Endothelium-Z000000PX GPL24676     Kidney tubular      tissue: Kidney tubular;\tcell type: Endothelium;\tlab: Had-HU;\tage: 65;\tSex: M;\trace: White
## 13    Kidney-Tubular-Endothelium-Z000000Q3 GPL24676     Kidney tubular      tissue: Kidney tubular;\tcell type: Endothelium;\tlab: Had-HU;\tage: 67;\tSex: M;\trace: White
## 14    Kidney-Tubular-Endothelium-Z0000042R GPL24676     Kidney tubular      tissue: Kidney tubular;\tcell type: Endothelium;\tlab: Had-HU;\tage: 74;\tSex: F;\trace: White
## 15             Liver-Endothelium-Z000000RB GPL24676              Liver                    tissue: Liver;\tcell type: Endothelium;\tlab: OHSU;\tage: 46;\tSex: F;\trace: NA
## 16     Lung-Alveolar-Endothelium-Z000000Q1 GPL24676      Lung alveolar       tissue: Lung alveolar;\tcell type: Endothelium;\tlab: Had-HU;\tage: 28;\tSex: F;\trace: White
## 17     Lung-Alveolar-Endothelium-Z000000QK GPL24676      Lung alveolar       tissue: Lung alveolar;\tcell type: Endothelium;\tlab: Had-HU;\tage: 76;\tSex: F;\trace: White
## 18     Lung-Alveolar-Endothelium-Z0000045H GPL24676      Lung alveolar       tissue: Lung alveolar;\tcell type: Endothelium;\tlab: Had-HU;\tage: 81;\tSex: F;\trace: White
## 19          Pancreas-Endothelium-Z0000042D GPL24676           Pancreas                tissue: Pancreas;\tcell type: Endothelium;\tlab: UA;\tage: 61;\tSex: F;\trace: White
## 20          Pancreas-Endothelium-Z0000042X GPL24676           Pancreas                tissue: Pancreas;\tcell type: Endothelium;\tlab: UA;\tage: 49;\tSex: F;\trace: White
## 21          Pancreas-Endothelium-Z00000430 GPL24676           Pancreas                tissue: Pancreas;\tcell type: Endothelium;\tlab: UA;\tage: 52;\tSex: F;\trace: White
## 22    Pancreas-Islet-Endothelium-Z0000042Y GPL24676           Pancreas                tissue: Pancreas;\tcell type: Endothelium;\tlab: UA;\tage: 52;\tSex: F;\trace: White
## 23             Colon-Fibroblasts-Z0000042A GPL24676              Colon                tissue: Colon;\tcell type: Fibroblast;\tlab: Had-HU;\tage: 37;\tSex: M;\trace: White
## 24             Colon-Fibroblasts-Z0000042C GPL24676              Colon                tissue: Colon;\tcell type: Fibroblast;\tlab: Had-HU;\tage: 46;\tSex: M;\trace: White
## 25             Heart-Fibroblasts-Z0000043R GPL24676              Heart                 tissue: Heart;\tcell type: Fibroblast;\tlab: Lonza;\tage: 55;\tSex: M;\trace: Black
## 26             Heart-Fibroblasts-Z0000041V GPL24676              Heart                 tissue: Heart;\tcell type: Fibroblast;\tlab: Lonza;\tage: 50;\tSex: F;\trace: White
## 27             Heart-Fibroblasts-Z0000041W GPL24676              Heart                 tissue: Heart;\tcell type: Fibroblast;\tlab: Lonza;\tage: 35;\tSex: F;\trace: White
## 28             Heart-Fibroblasts-Z0000041X GPL24676              Heart                 tissue: Heart;\tcell type: Fibroblast;\tlab: Lonza;\tage: 61;\tSex: M;\trace: White
## 29            Dermal-Fibroblasts-Z00000423 GPL24676          Ad-dermal             tissue: Ad-dermal;\tcell type: Fibroblast;\tlab: Lonza;\tage: 73;\tSex: M;\trace: White
## 30               Skeletal-Muscle-Z00000427 GPL24676    Skeletal muscle tissue: Skeletal muscle;\tcell type: Striated muscle;\tlab: Had-HU;\tage: 59;\tSex: F;\trace: White
## 31               Skeletal-Muscle-Z00000429 GPL24676    Skeletal muscle tissue: Skeletal muscle;\tcell type: Striated muscle;\tlab: Had-HU;\tage: 34;\tSex: M;\trace: White
## 32           Aorta-Smooth-Muscle-Z0000041U GPL24676              Aorta              tissue: Aorta;\tcell type: Smooth muscle;\tlab: Lonza;\tage: 22;\tSex: M;\trace: White
## 33 Coronary-Artery-Smooth-Muscle-Z00000420 GPL24676     Cornary artery     tissue: Cornary artery;\tcell type: Smooth muscle;\tlab: Lonza;\tage: 17;\tSex: F;\trace: White
## 34         Bladder-Smooth-Muscle-Z0000041Z GPL24676            Bladder            tissue: Bladder;\tcell type: Smooth muscle;\tlab: Lonza;\tage: 51;\tSex: F;\trace: Black
## 35        Prostate-Smooth-Muscle-Z0000041Y GPL24676           Prostate           tissue: Prostate;\tcell type: Smooth muscle;\tlab: Lonza;\tage: 26;\tSex: M;\trace: White
## 36   Lung-Bronchus-Smooth-Muscle-Z00000421 GPL24676          Bronchial           tissue: Bronchial;\tcell type: Smooth muscle;\tlab: Lonza;\tage: 57;\tSex: M;\trace: Hisp
## 37           Heart-Cardiomyocyte-Z0000044G GPL24676              Heart             tissue: Heart;\tcell type: Cardiomyocyte;\tlab: Had-HU;\tage: 83;\tSex: M;\trace: White
## 38           Heart-Cardiomyocyte-Z0000044K GPL24676              Heart             tissue: Heart;\tcell type: Cardiomyocyte;\tlab: Had-HU;\tage: 54;\tSex: M;\trace: White
## 39           Heart-Cardiomyocyte-Z0000044N GPL24676              Heart             tissue: Heart;\tcell type: Cardiomyocyte;\tlab: Had-HU;\tage: 59;\tSex: M;\trace: White
## 40           Heart-Cardiomyocyte-Z0000044P GPL24676              Heart             tissue: Heart;\tcell type: Cardiomyocyte;\tlab: Had-HU;\tage: 71;\tSex: F;\trace: White
## 41           Heart-Cardiomyocyte-Z0000044Q GPL24676              Heart             tissue: Heart;\tcell type: Cardiomyocyte;\tlab: Had-HU;\tage: 81;\tSex: M;\trace: White
## 42           Heart-Cardiomyocyte-Z0000044R GPL24676              Heart             tissue: Heart;\tcell type: Cardiomyocyte;\tlab: Had-HU;\tage: 70;\tSex: M;\trace: White
## 43              Bone-Osteoblasts-Z0000042Z GPL24676               Bone                   tissue: Bone;\tcell type: Osteoblasts;\tlab: Lonza;\tage: 5;\tSex: F;\trace: Hisp
## 44              Oligodendrocytes-Z000000TK GPL24676              Brain                 tissue: Brain;\tcell type: Oligodendrocytes;\tlab: KI;\tage: 31;\tSex: M;\trace: NA
## 45              Oligodendrocytes-Z0000042E GPL24676              Brain                 tissue: Brain;\tcell type: Oligodendrocytes;\tlab: KI;\tage: 56;\tSex: M;\trace: NA
## 46              Oligodendrocytes-Z0000042L GPL24676              Brain                 tissue: Brain;\tcell type: Oligodendrocytes;\tlab: KI;\tage: 25;\tSex: M;\trace: NA
## 47              Oligodendrocytes-Z0000042N GPL24676              Brain                 tissue: Brain;\tcell type: Oligodendrocytes;\tlab: KI;\tage: 39;\tSex: M;\trace: NA
## 48                 Cortex-Neuron-Z000000TF GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 38;\tSex: M;\trace: NA
## 49                        Neuron-Z000000TH GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 31;\tSex: M;\trace: NA
## 50                 Cortex-Neuron-Z0000042F GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 56;\tSex: M;\trace: NA
## 51                 Cortex-Neuron-Z0000042H GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 65;\tSex: F;\trace: NA
## 52                 Cortex-Neuron-Z0000042J GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 25;\tSex: M;\trace: NA
## 53                 Cortex-Neuron-Z0000042M GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 25;\tSex: M;\trace: NA
## 54                 Cortex-Neuron-Z0000042P GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 39;\tSex: M;\trace: NA
## 55                 Cortex-Neuron-Z0000042K GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 25;\tSex: M;\trace: NA
## 56             Cerebellum-Neuron-Z000000TB GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 19;\tSex: M;\trace: NA
## 57                 Cortex-Neuron-Z000000TD GPL24676              Brain                         tissue: Brain;\tcell type: Neuronal;\tlab: KI;\tage: 19;\tSex: M;\trace: NA
## 58             Liver-Hepatocytes-Z000000R3 GPL24676              Liver                     tissue: Liver;\tcell type: Hepatocyte;\tlab: OHSU;\tage: 24;\tSex: M;\trace: NA
## 59             Liver-Hepatocytes-Z000000T3 GPL24676              Liver                    tissue: Liver;\tcell type: Hepatocyte;\tlab: OHSU;\tage: NA;\tSex: NA;\trace: NA
## 60             Liver-Hepatocytes-Z0000043Q GPL24676              Liver                 tissue: Liver;\tcell type: Hepatocyte;\tlab: Lonza;\tage: 56;\tSex: F;\trace: White
## 61             Liver-Hepatocytes-Z0000044H GPL24676              Liver                tissue: Liver;\tcell type: Hepatocyte;\tlab: Had-HU;\tage: 55;\tSex: M;\trace: White
## 62             Liver-Hepatocytes-Z0000044M GPL24676              Liver                tissue: Liver;\tcell type: Hepatocyte;\tlab: Had-HU;\tage: 66;\tSex: M;\trace: White
## 63             Liver-Hepatocytes-Z00000431 GPL24676              Liver                 tissue: Liver;\tcell type: Hepatocyte;\tlab: Lonza;\tage: 45;\tSex: F;\trace: White
## 64                 Pancreas-Duct-Z0000043T GPL24676           Pancreas                          tissue: Pancreas;\tcell type: Duct;\tlab: UA;\tage: 26;\tSex: F;\trace: NA
## 65                 Pancreas-Duct-Z0000043U GPL24676           Pancreas                          tissue: Pancreas;\tcell type: Duct;\tlab: UA;\tage: 19;\tSex: M;\trace: NA
## 66                 Pancreas-Duct-Z0000043V GPL24676           Pancreas                          tissue: Pancreas;\tcell type: Duct;\tlab: UA;\tage: 23;\tSex: M;\trace: NA
## 67                 Pancreas-Duct-Z000000QZ GPL24676           Pancreas                        tissue: Pancreas;\tcell type: Duct;\tlab: OHSU;\tage: 45;\tSex: M;\trace: NA
## 68               Pancreas-Acinar-Z000000QX GPL24676           Pancreas                      tissue: Pancreas;\tcell type: Acinar;\tlab: OHSU;\tage: 45;\tSex: M;\trace: NA
## 69               Pancreas-Acinar-Z0000043W GPL24676           Pancreas                        tissue: Pancreas;\tcell type: Acinar;\tlab: UA;\tage: 26;\tSex: F;\trace: NA
## 70               Pancreas-Acinar-Z0000043X GPL24676           Pancreas                        tissue: Pancreas;\tcell type: Acinar;\tlab: UA;\tage: 19;\tSex: M;\trace: NA
## 71               Pancreas-Acinar-Z0000043Y GPL24676           Pancreas                        tissue: Pancreas;\tcell type: Acinar;\tlab: UA;\tage: 23;\tSex: M;\trace: NA
## 72                Pancreas-Delta-Z00000451 GPL24676           Pancreas                         tissue: Pancreas;\tcell type: Delta;\tlab: UA;\tage: 58;\tSex: F;\trace: NA
## 73                Pancreas-Delta-Z00000454 GPL24676           Pancreas                         tissue: Pancreas;\tcell type: Delta;\tlab: UA;\tage: 43;\tSex: F;\trace: NA
## 74                Pancreas-Delta-Z00000457 GPL24676           Pancreas                    tissue: Pancreas;\tcell type: Delta;\tlab: IIDP;\tage: 58;\tSex: F;\trace: White
## 75                 Pancreas-Beta-Z00000452 GPL24676           Pancreas                          tissue: Pancreas;\tcell type: Beta;\tlab: UA;\tage: 58;\tSex: F;\trace: NA
## 76                 Pancreas-Beta-Z00000455 GPL24676           Pancreas                          tissue: Pancreas;\tcell type: Beta;\tlab: UA;\tage: 43;\tSex: F;\trace: NA
## 77                 Pancreas-Beta-Z00000458 GPL24676           Pancreas                     tissue: Pancreas;\tcell type: Beta;\tlab: IIDP;\tage: 58;\tSex: F;\trace: White
## 78                Pancreas-Alpha-Z00000453 GPL24676           Pancreas                         tissue: Pancreas;\tcell type: Alpha;\tlab: UA;\tage: 58;\tSex: F;\trace: NA
## 79                Pancreas-Alpha-Z00000456 GPL24676           Pancreas                         tissue: Pancreas;\tcell type: Alpha;\tlab: UA;\tage: 43;\tSex: F;\trace: NA
## 80                Pancreas-Alpha-Z00000459 GPL24676           Pancreas                    tissue: Pancreas;\tcell type: Alpha;\tlab: IIDP;\tage: 58;\tSex: F;\trace: White
## 81  Kidney-Glomerular-Epithelium-Z0000045K GPL24676  Kidney glomerular    tissue: Kidney glomerular;\tcell type: Epithelium;\tlab: Had-HU;\tage: 47;\tSex: M;\trace: White
## 82  Kidney-Glomerular-Epithelium-Z0000045L GPL24676  Kidney glomerular    tissue: Kidney glomerular;\tcell type: Epithelium;\tlab: Had-HU;\tage: 47;\tSex: M;\trace: White
## 83     Kidney-Tubular-Epithelium-Z000000QH GPL24676     Kidney tubular       tissue: Kidney tubular;\tcell type: Epithelium;\tlab: Had-HU;\tage: 47;\tSex: M;\trace: White
## 84     Kidney-Tubular-Epithelium-Z0000043Z GPL24676     Kidney tubular       tissue: Kidney tubular;\tcell type: Epithelium;\tlab: Had-HU;\tage: 60;\tSex: M;\trace: White
## 85     Kidney-Tubular-Epithelium-Z00000440 GPL24676     Kidney tubular       tissue: Kidney tubular;\tcell type: Epithelium;\tlab: Had-HU;\tage: 60;\tSex: M;\trace: White
## 86   Kidney-Glomerular-Podocytes-Z0000042W GPL24676  Kidney glomerular      tissue: Kidney glomerular;\tcell type: Podocyte;\tlab: Had-HU;\tage: 74;\tSex: F;\trace: White
## 87   Kidney-Glomerular-Podocytes-Z00000441 GPL24676  Kidney glomerular      tissue: Kidney glomerular;\tcell type: Podocyte;\tlab: Had-HU;\tage: 60;\tSex: M;\trace: White
## 88   Kidney-Glomerular-Podocytes-Z00000442 GPL24676  Kidney glomerular      tissue: Kidney glomerular;\tcell type: Podocyte;\tlab: Had-HU;\tage: 60;\tSex: M;\trace: White
## 89            Thyroid-Epithelium-Z0000042S GPL24676            Thyroid              tissue: Thyroid;\tcell type: Epithelium;\tlab: Had-HU;\tage: 28;\tSex: F;\trace: White
## 90            Thyroid-Epithelium-Z0000042T GPL24676            Thyroid              tissue: Thyroid;\tcell type: Epithelium;\tlab: Had-HU;\tage: 40;\tSex: F;\trace: White
##                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       supplementary_file
## 1  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 2  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 3  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 4  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 5  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 6  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 7  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 8  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 9  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 10 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 11 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 12 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 13 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 14 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 15 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 16 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 17 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 18 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 19 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 20 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 21 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 22 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 23 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 24 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 25 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 26 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 27 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 28 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 29 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 30 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 31 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 32 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 33 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 34 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 35 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 36 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 37 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 38 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 39 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 40 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 41 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 42 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 43 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 44 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 45 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 46 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 47 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 48 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 49 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 50 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 51 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 52 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 53 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 54 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 55 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 56 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 57 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 58 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 59 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 60 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 61 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 62 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 63 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 64 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 65 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 66 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 67 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 68 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 69 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 70 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 71 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 72 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 73 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 74 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 75 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 76 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 77 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 78 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 79 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 80 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 81 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 82 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 83 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 84 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 85 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 86 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 87 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 88 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 89 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
## 90 ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_RAW.tar;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s205.bed.gz.tbi.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz;\tftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE186nnn/GSE186458/suppl/GSE186458_blocks.s207.hg38.bed.gz.tbi.gz
##  [ reached &#39;max&#39; / getOption(&quot;max.print&quot;) -- omitted 163 rows ]</code></pre>
<ol style="list-style-type: decimal">
<li>The GSE186458 study is high throughput sequencing of methylation profiles. Specifically looking at DNA methylation patterns in adults with the use of deep whole-genome bisulfite sequencing.</li>
<li>It is not using a disease, the study models the human methylome to gain a better understanding of regulatory elements within specific cell types.</li>
<li>There are 253 samples in this study with bed files containing the methylation signal provided per sample.</li>
<li>There is not a distinction between control and disease in this study, only in tissue-region as samples are collected throughout the body for example aorta and lung tissues.</li>
</ol>
<p><strong>GSE264108</strong></p>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="exercises-inspect-specific-geo-series.html#cb41-1" tabindex="-1"></a>sql <span class="ot">&lt;-</span> <span class="fu">paste</span>(</span>
<span id="cb41-2"><a href="exercises-inspect-specific-geo-series.html#cb41-2" tabindex="-1"></a>  <span class="st">&quot;SELECT DISTINCT gse.gse, gse.overall_design, gse.type, gsm.type, gse.title, gse.summary, gsm.title, gsm.gpl, gsm.source_name_ch1, gsm.characteristics_ch1, gse.supplementary_file&quot;</span>,</span>
<span id="cb41-3"><a href="exercises-inspect-specific-geo-series.html#cb41-3" tabindex="-1"></a>  <span class="st">&quot;FROM gse JOIN gse_gsm ON gse_gsm.gse = gse.gse&quot;</span>,</span>
<span id="cb41-4"><a href="exercises-inspect-specific-geo-series.html#cb41-4" tabindex="-1"></a>  <span class="st">&quot;JOIN gsm ON gse_gsm.gsm = gsm.gsm&quot;</span>,</span>
<span id="cb41-5"><a href="exercises-inspect-specific-geo-series.html#cb41-5" tabindex="-1"></a>  <span class="st">&quot;WHERE gse_gsm.gse = &#39;GSE264108&#39;&quot;</span></span>
<span id="cb41-6"><a href="exercises-inspect-specific-geo-series.html#cb41-6" tabindex="-1"></a>)</span>
<span id="cb41-7"><a href="exercises-inspect-specific-geo-series.html#cb41-7" tabindex="-1"></a><span class="fu">dbGetQuery</span>(con, sql)</span></code></pre></div>
<pre><code>##  [1] gse                 overall_design      type                type                title               summary             title               gpl                
##  [9] source_name_ch1     characteristics_ch1 supplementary_file 
## &lt;0 rows&gt; (or 0-length row.names)</code></pre>
<div class="sourceCode" id="cb43"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb43-1"><a href="exercises-inspect-specific-geo-series.html#cb43-1" tabindex="-1"></a>sql <span class="ot">&lt;-</span> <span class="fu">paste</span>(</span>
<span id="cb43-2"><a href="exercises-inspect-specific-geo-series.html#cb43-2" tabindex="-1"></a>  <span class="st">&quot;SELECT DISTINCT gse.gse, gse.overall_design, gse.type, gse.supplementary_file&quot;</span>,</span>
<span id="cb43-3"><a href="exercises-inspect-specific-geo-series.html#cb43-3" tabindex="-1"></a>  <span class="st">&quot;FROM gse&quot;</span>,</span>
<span id="cb43-4"><a href="exercises-inspect-specific-geo-series.html#cb43-4" tabindex="-1"></a>  <span class="st">&quot;WHERE gse.gse = &#39;GSE264108&#39;&quot;</span></span>
<span id="cb43-5"><a href="exercises-inspect-specific-geo-series.html#cb43-5" tabindex="-1"></a>)</span>
<span id="cb43-6"><a href="exercises-inspect-specific-geo-series.html#cb43-6" tabindex="-1"></a><span class="fu">dbGetQuery</span>(con, sql)</span></code></pre></div>
<pre><code>## [1] gse                overall_design     type               supplementary_file
## &lt;0 rows&gt; (or 0-length row.names)</code></pre>
<p>The GEO data set GSE264108 has no information present as the dataset is not currently listed in the active database, it has potentially been retired or is no longer accessible.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="finding-datasets-of-interest.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="next-week-next-steps.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": null,
    "text": null
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": null,
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

</body>

</html>
